pre-docs/writing_yaml_plans.md

```diff
@@ -473,3 +473,134 @@ plan mymodule::yamlplan(
 }
 ```
 
+## YAML plan constructs which cannot be translated to Puppet plans
```

Comment:
Super nit-picky: because this is a defining clause it should be `YAML plan constructs that cannot...`

Reply:
Not nitpicky to be grammatically correct.